### PR TITLE
[schema-registry] ref: custom registry UI image + tweak config

### PR DESCRIPTION
**Pull request by Pinimo**
Related to https://blablacar.atlassian.net/browse/DBRE-3398

### CODE REVIEW

**Review by frivoire — COMMENTED** [review]
Do you really need to replac the old "env variables" method to configure it ?
It looks much simpler and cleaner "interface" between K8S and the docker image 😕

**Review by OuesFa — APPROVED** [review]

File: charts/cp-schema-registry/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+## cp-schema-registry-v7.3.1-bbc-2
+
+* Schema registry bugfix:
+    * chore: Upgraded Caddy server to 2.6.4 in the Schema Registry UI
+    * ref: Standardized configuration with a hard-coded config in the Docker image, no
+      entrypoint
+    * fix: `Oidc_*` Headers are not proxied anymore to the backend Schema
+      Registry. This reduces a lot the total header length, avoiding hitting
+      the max header length limit on the backend side.
```
Comment (frivoire) [inline charts/cp-schema-registry/CHANGELOG.md:11]: This chart is only for the schema-registry, so you can just list changes "directly", without a subsection:
```suggestion
* chore: Upgraded Caddy server to 2.6.4 for the UI
* ref: Standardized configuration with a hard-coded config in the Docker image, no
   entrypoint
* fix: `Oidc_*` Headers are not proxied anymore to the backend Schema
   Registry. This reduces a lot the total header length, avoiding hitting
   the max header length limit on the backend side.
```

Comment (frivoire) [reply charts/cp-schema-registry/CHANGELOG.md:11]: BTW: what do you call "ref" ?

**Review by frivoire — APPROVED** [review]
👍

File: charts/cp-schema-registry/CHANGELOG.md (hunk 1 above)
Comment (frivoire) [reply charts/cp-schema-registry/CHANGELOG.md]: Ha :) :) :)
Thanks !

**Pull request merged**